src/genie/libs/parser/iosxr/tests/ShowRibTablesSummary/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,12 @@
+expected_output = {
+                'rib_summary': {
+                        'table_id': {
```

Comment:
just naming. `rib_table` might be better for value `ipv4` or `ipv6`.
```python
{
    'rib_table': {
        'ipv4': {
            'num_multicast_tables': 1,
(snip)

Reply:
Agree with you, that would make it a little clearer. I've made that change as well.